app/assets/images/caretDown.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M5 8l5 5 5-5" fill="#248AF2" fill-rule="evenodd"/></svg>
```

Comment:
All other images use `lower-case-with-dashes` for names. Can you rename that file please for consistency.